src/display/api.js

```diff
@@ -1010,13 +1010,17 @@ class PDFDocumentProxy {
  *   whitespace with standard spaces (0x20). The default value is `false`.
  * @property {boolean} disableCombineTextItems - Do not attempt to combine
  *   same line {@link TextItem}'s. The default value is `false`.
+ * @property {boolean} includeMarkedContent - When true include marked
```

Comment:
To tell JSDocs that the property is optional, please make the following change (it looks like we forgot to fix that for the ones above):
```suggestion
 * @property {boolean} [includeMarkedContent] - When true include marked
```